enzyme/Enzyme/InstructionDerivatives.td

```diff
@@ -350,6 +366,33 @@ def : CallPattern<(Op $x),
                   (ForwardFromSummedReverse),
                   [ReadNone, NoUnwind]
                   >;
+def : CallPattern<(Op $x),
+                  ["sinhl"],
+                  [(FMul (DiffeRet), (Call<(SameTypesFunc<"coshl">), [ReadNone,NoUnwind]> $x))],
+                  (ForwardFromSummedReverse),
+                  [ReadNone, NoUnwind]
+                  >;
+
+def : CallPattern<(Op $x),
+                  ["asinh", "asinhf", "asinhl", "__nv_asinh", "__nv_asinhf"],
```

Comment:
can you add tests for asinh/acosh/atanh?

Reply:
Right, [4001e41](https://github.com/EnzymeAD/Enzyme/pull/1605/commits/4001e41d9780681159cc473d638a97a11c659bde) adds tests for asinh, asinhf, asinhl, acosh, atanh in the forward and reverse mode.

There are also no tests so far for the error function and nearbyint. Is this ok or should I add tests for those?